### ISSUE TITLE
Reorder Frames Error

### ISSUE THREAD

**Issue by BobbyOfWindsor1974** [fuel9/DisplayMonkey#170]
When trying to reorder frames, I get this error as soon as I click the reorder frames link on the MAIN panel.

Source | Panel/SortFrames
-- | --
Message | Object reference not set to an instance of an object.
Stack trace | at ASP._Page_Views_Video__Film_cshtml.Execute() in c:\inetpub\wwwroot\DisplayMonkey_Management\Views\Video\_Film.cshtml:line 8    at System.Web.WebPages.WebPageBase.ExecutePageHierarchy()    at System.Web.Mvc.WebViewPage.ExecutePageHierarchy()    at System.Web.WebPages.WebPageBase.ExecutePageHierarchy(WebPageContext pageContext, TextWriter writer, WebPageRenderingBase startPage)    at System.Web.Mvc.RazorView.RenderView(ViewContext viewContext, TextWriter writer, Object instance)    at System.Web.Mvc.BuildManagerCompiledView.Render(ViewContext viewContext, TextWriter writer)    at System.Web.Mvc.HtmlHelper.RenderPartialInternal(String partialViewName, ViewDataDictionary viewData, Object model, TextWriter writer, ViewEngineCollection viewEngineCollection)    at System.Web.Mvc.Html.PartialExtensions.Partial(HtmlHelper htmlHelper, String partialViewName, Object model, ViewDataDictionary viewData)    at System.Web.Mvc.Html.PartialExtensions.Partial(HtmlHelper htmlHelper, String partialViewName, Object model)    at ASP._Page_Views_Panel_SortFrames_cshtml.Execute() in c:\inetpub\wwwroot\DisplayMonkey_Management\Views\Panel\SortFrames.cshtml:line 48    at System.Web.WebPages.WebPageBase.ExecutePageHierarchy()    at System.Web.Mvc.WebViewPage.ExecutePageHierarchy()

**Comment by michaelmalmgren** [comment]
Hi Bobby,

I tried to reproduce this but I was not able to. Once you click on reorder frames this page should be displayed:
![image](https://user-images.githubusercontent.com/25671627/75329277-aa54da80-587f-11ea-90ca-fe30e9a7f798.png)

I assume that is where you get the error?
What types of frames should have been displayed?

**Comment by BobbyOfWindsor1974** [comment]
That is the one I see for every other fame.  Just not this one. We have a mix of pics, videos, etc.

Is there away to move them to another from the DB?

I want to create a new frame..then move the items off one by one to see if I can narrow it down to an entry?




Thanks

--Bobby



Robert Hicks
Senior Technical Analyst

Leggett & Platt Automotive
360 Silvercreek Industrial Dr., Lakeshore, ON N8N 4Y3
Office: +1 519 727-7000 ext. 1003

bobby.hicks@leggett.com<mailto:bobby.hicks@leggett.com>
Leggett-Automotive.com
[Description: Description: blue facebook 18]<https://www.facebook.com/LeggettAuto/> [Description: Description: blue insta 18] <https://www.instagram.com/leggettauto/>  [Description: Description: blue linkedin 18] <https://www.linkedin.com/company/leggettauto/>  [Description: Description: blue twitter 18] <https://twitter.com/LeggettAuto>

[Description: Description: L&P_Automotive_Group_w-Tagline_email]
P Please consider your environmental responsibility before printing this e-mail

**Comment by michaelmalmgren** [comment]
Yes, you can change the sort or move a frame from one panel to another in the table
[dbo].[Frame]. PanelId is equal to the number you see in the URL, in my example above it is 1005 ie. select * from [dbo].[Frame] where PanelId = 1005

**Comment by BobbyOfWindsor1974** [comment]
Thanks. I will attempt to recreate the frame and add things back one at a time to see if something breaks.  I’ll keep you posted.




Thanks

--Bobby



Robert Hicks
Senior Technical Analyst

Leggett & Platt Automotive
360 Silvercreek Industrial Dr., Lakeshore, ON N8N 4Y3
Office: +1 519 727-7000 ext. 1003

bobby.hicks@leggett.com<mailto:bobby.hicks@leggett.com>
Leggett-Automotive.com
[Description: Description: blue facebook 18]<https://www.facebook.com/LeggettAuto/> [Description: Description: blue insta 18] <https://www.instagram.com/leggettauto/>  [Description: Description: blue linkedin 18] <https://www.linkedin.com/company/leggettauto/>  [Description: Description: blue twitter 18] <https://twitter.com/LeggettAuto>

[Description: Description: L&P_Automotive_Group_w-Tagline_email]
P Please consider your environmental responsibility before printing this e-mail

**Comment by BobbyOfWindsor1974** [comment]
Thanks. I found the two slides with the issue. I could not determine what the issue was, but once removed, the re-order worked again.




Thanks

--Bobby



Robert Hicks
Senior Technical Analyst

Leggett & Platt Automotive
360 Silvercreek Industrial Dr., Lakeshore, ON N8N 4Y3
Office: +1 519 727-7000 ext. 1003

bobby.hicks@leggett.com<mailto:bobby.hicks@leggett.com>
Leggett-Automotive.com
[Description: Description: blue facebook 18]<https://www.facebook.com/LeggettAuto/> [Description: Description: blue insta 18] <https://www.instagram.com/leggettauto/>  [Description: Description: blue linkedin 18] <https://www.linkedin.com/company/leggettauto/>  [Description: Description: blue twitter 18] <https://twitter.com/LeggettAuto>

[Description: Description: L&P_Automotive_Group_w-Tagline_email]
P Please consider your environmental responsibility before printing this e-mail
